standalone/standalone.go

```diff
@@ -99,6 +99,8 @@ func Handler(ch grpcdynamic.Channel, target string, methods []*desc.MethodDescri
 	rpcMetadataHandler := grpcui.RPCMetadataHandler(methods, files)
 	mux.Handle("/metadata", rpcMetadataHandler)
 
+	registerExamplesHandler(&mux, uiOpts)
```

Comment:
While I understand this makes the code more modular and more concise, it's also inconsistent with the rest of this method -- you've moved the handler out to a separate function even though it's no more complicated/no larger than any of the other handlers inlined into this function. Consider inlining.